crates/pyrefly_types/src/callable.rs

```diff
@@ -334,6 +334,8 @@ pub struct FuncFlags {
     pub has_final_decoration: bool,
     /// A function decorated with `@abc.abstractmethod`
     pub is_abstract_method: bool,
+    /// Function body is treated as a stub (e.g. body is `...` or absent in a stub file)
+    pub is_stub: bool,
```

Comment:
Could we change this to `lacks_implementation`?

The current name isn't terrible, but the problem problem with `is_stub` is that given standard "a function is a stub" naively means "is it defined in a `.pyi` file, which is not what this flag represents